libcontainer/dmz/_dmz.c

```diff
@@ -11,5 +13,17 @@ int main(int argc, char **argv)
 {
 	if (argc < 1)
 		return 127;
-	return execve(argv[0], argv, environ);
+	int r = execve(argv[0], argv, environ);
```

Comment:
`int ret`

Reply:
Heh, I was doubting if it was worth to push again for that. But now that you mention it, no doubt! :)